src/compiler/generate-xspec-tests.xsl

```diff
@@ -87,6 +87,7 @@
     </template>
     <!-- Compile the top-level scenarios. -->
     <xsl:call-template name="x:compile-scenarios"/>
+    <xsl:copy-of select="document('')/xsl:stylesheet/xsl:function[contains(@name, 'schematron-location-')]"/>
```

Comment:
Just curious. Why embedding the functions directly rather than creating `xsl:import`?